core/src/main/scala/kafka/network/SocketServer.scala

```diff
@@ -88,25 +94,35 @@ class SocketServer(val config: KafkaConfig, val metrics: Metrics, val time: Time
   def startup(startupProcessors: Boolean = true) {
     this.synchronized {
       connectionQuotas = new ConnectionQuotas(config.maxConnectionsPerIp, config.maxConnectionsPerIpOverrides)
-      createAcceptorAndProcessors(config.numNetworkThreads, config.listeners)
+      createAcceptorAndProcessors(config.numNetworkThreads, config.dataListeners, dataRequestChannel, dataProcessors, false)
```

Comment:
Was thinking if we should have 2 explicit functions : 

- createDataPlaneAcceptorsAndProcessors(processorsPerListener : Int, endpoints: Seq[EndPoint])

- createControlPlaneAcceptorAndProcessor ()

We can probably do something like : 
```suggestion
  createDataPlaneAcceptorsAndProcessors(config.numNetworkThreads, config.dataListeners)
  createControlPlaneAcceptorAndProcessor()
```